opentracing/src/main/java/opentracing/propagation/SplitTextFormatInjector.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2016 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package opentracing.propagation;
+
+import java.util.Map;
+
+/**
+ *
+ */
+interface SplitTextFormatInjector extends Injector<Map> {
```

Comment:
> owners of said distributed systems should be able to strip out baggage without requiring changes to the OT implementation they're using

The dropping of baggage is an operational optimisation, a safeguard, not something that's intrinsic part of the public api.

Furthermore i believe that it's something that otherwise anyway can be done in a more elegant, non-intrusive to the public api, manner.

For example look at how the  [SplitTextFormatInjectorImpl.java](https://github.com/opentracing/opentracing-java/pull/11/files#diff-f2ad9c00db83aa2576fe12ea2f7f0413) solves it.